### PR TITLE
:zap: template for model, controller, tests

### PR DESCRIPTION
**Pull request by fedoranvar**

### CODE REVIEW

File: bobtemplates/itpp/odoo/wizard/wizards/+wizard.name_underscored+.xml.bob [hunk 1]
```diff
@@ -1,86 +0,0 @@
...
-        <field name="inherit_id" ref="TODO othermodule.wizard"/>
-{{% endif %}}
-        <field name="arch" type="xml">
-{{% if not wizard.inherit %}}
-            <form string="{{{ wizard.name_camelwords }}}">
-                <!-- TODO -->
-                <group>
-                    <field name="name"/>
-                </group>
-                <footer>
```
Comment (KolushovAlexandr) [inline bobtemplates/itpp/odoo/wizard/wizards/+wizard.name_underscored+.xml.bob:26]: Restore group and footer is a good idea, I did not know it was already done.

File: bobtemplates/itpp/odoo/wizard/wizards/+wizard.name_underscored+.xml.bob [hunk 2]
```diff
@@ -1,86 +0,0 @@
...
-            </form>
-{{% else %}}
-            <!-- TODO -->
-{{% endif %}}
-        </field>
-    </record>
-{{% endif %}}
-
-{{% if wizard.view_action %}}
-    <record model="ir.actions.act_window" id="{{{ wizard.name_underscored }}}_act_window">
```
Comment (KolushovAlexandr) [inline bobtemplates/itpp/odoo/wizard/wizards/+wizard.name_underscored+.xml.bob:55]: and this one too

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]
Add empty lines at the end of files where it's not done yet.

**Comment by fedoranvar** [comment]
> Add empty lines at the end of files where it's not done yet.

Haven't found any file that doesn't have an empty line in EOF

File: setup.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 setup(
     name="bobtemplates.odoo",
     use_scm_version=True,
-    description="mr.bob templates for Odoo projects",
+    description="mr.bob templates for Odoo projects (IT-Projects Edition",
```
Comment (KolushovAlexandr) [inline setup.py]:
```suggestion
    description="mr.bob templates for Odoo projects (IT-Projects Edition)",
```

File: bobtemplates/itpp/odoo/view/.mrbob.ini [hunk 4]
```diff
@@ -0,0 +1,9 @@
+[questions]
+view.name_underscored.question = View name (underscored notation)
+view.name_underscored.required = True
+view.name_underscored.post_ask_question = mrbob.hooks:validate_regex
+view.name_underscored.regex = ^[a-z_]+$
+
+[template]
+pre_render = bobtemplates.itpp.odoo.hooks:pre_render_view
+post_render = bobtemplates.itpp.odoo.hooks:post_render_view
```
Comment (KolushovAlexandr) [inline bobtemplates/itpp/odoo/view/.mrbob.ini]:
```suggestion
post_render = bobtemplates.itpp.odoo.hooks:post_render_view

```

File: bobtemplates/itpp/odoo/test/static/src/js/test_+test.name_underscored+.js.bob [hunk 5]
```diff
@@ -0,0 +1,24 @@
+/*  Copyright {{{ copyright.year }}} {{{ copyright.name }}} <https://it-projects.info/team/{{{ copyright.github }}}>
+    License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html). */
+
```
Comment (KolushovAlexandr) [inline bobtemplates/itpp/odoo/test/static/src/js/test_+test.name_underscored+.js.bob]:
```suggestion
```

File: bobtemplates/itpp/odoo/test/.mrbob.ini [hunk 6]
```diff
@@ -1,21 +1,17 @@
...
-copyright.year.required = True
+test.tour.question = JS-tour (phantom_js) (y or n) ?
+test.tour.post_ask_question = mrbob.hooks:to_boolean
+test.tour.required = True
 
 [template]
-pre_render = bobtemplates.odoo.hooks:pre_render_test
-post_render = bobtemplates.odoo.hooks:post_render_test
+pre_render = bobtemplates.itpp.odoo.hooks:pre_render_test
+post_render = bobtemplates.itpp.odoo.hooks:post_render_test
```
Comment (KolushovAlexandr) [inline bobtemplates/itpp/odoo/test/.mrbob.ini]:
```suggestion
post_render = bobtemplates.itpp.odoo.hooks:post_render_test

```

File: bobtemplates/itpp/odoo/controller/controllers/+controller.name_underscored+.py.bob [hunk 7]
```diff
@@ -0,0 +1,12 @@
+{{% if odoo.version < 11 %}}# -*- coding: utf-8 -*-
+{{% endif %}}
+# Copyright {{{ copyright.year }}} {{{ copyright.name }}} <https://it-projects.info/team/{{{ copyright.github }}}>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
```
Comment (KolushovAlexandr) [inline bobtemplates/itpp/odoo/controller/controllers/+controller.name_underscored+.py.bob:4]:
```suggestion
# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).

```

File: README.rst [hunk 8]
```diff
@@ -1,53 +1,104 @@
...
+  * ``addon`` : Directory with minimum files, that allows to install addon in odoo
+  * ``controller`` : Controller that prints "Hello World" by the url ``{ODOO_ROOT_URL}/hello/world``
+  * ``css`` : Empty ``.css``
+  * ``data`` : Empty ``.xml``
+  * ``demo`` : Empty ``.xml``
+  * ``js`` : ``.js``-file and ``.xml``-file for applying JS
+  * ``qweb`` : Empty ``.xml``-file in ``{ADDON_DIRECTORY}/static/src/xml``-directory
+  * ``test`` : ``.py``-file in ``tests`` directory
+  * ``view`` : Empty ``.xml``-file in ``views``-directory
+  * ``wizard`` :
```
Comment (KolushovAlexandr) [inline README.rst:62]: add description here

File: README.rst [hunk 9]
```diff
@@ -1,53 +1,104 @@
...
-  * ``model``: an Odoo model with accompanying form, tree, action, menu,
-    demo data and ACL
-  * ``test``: a test class
-  * ``wizard``: a wizard with transient model, view and action
+Configuration
+~~~~~~~~~~~~~
 
-Install
-~~~~~~~
+* Set copyright variables in ``.mrbob.ini`` in ``/path/to/config``:
```
Comment (KolushovAlexandr) [inline README.rst]: There is no any link to available variables, what should i write there?

File: README.rst [hunk 10]
```diff
@@ -1,53 +1,104 @@
...
 
-* To create addon-template in current directory (non-interactive):
+    mrbob ... -O /path/to/directory
 
+* To set default values of variables put code in ``/path/to/config/.mrbob.ini``:
+  
   .. code:: shell
 
-    mrbob bobtemplates.itpp.odoo:addon -n
+      [variables]
```
Comment (KolushovAlexandr) [inline README.rst]: there was a notice about it somewhere above, and here it is again. But still, there is no proper example of available variables. Not sure that putting a version here is a good idea

File: README.rst [hunk 11]
```diff
@@ -1,53 +1,104 @@
...
 
-    mrbob -O /path/to/directory" bobtemplates.itpp.odoo:addon -n
+    ### (default values of variables)
+
+    mrbob ... -n
 
 Useful links
 ~~~~~~~~~~~~
 
-* mr.bob readthedocs: https://mrbob.readthedocs.io/en/latest/index.html
```
Comment (KolushovAlexandr) [inline README.rst]: why this one was deleted?

File: bobtemplates/itpp/copyright/.mrbob.ini [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+copyright.name.question = Copyright name
+copyright.name.required = True
+copyright.name.post_ask_question = mrbob.hooks:validate_regex
+copyright.name.regex = ^[A-Za-z\s]+$
+
+copyright.github.question = Copyright github-name
+copyright.github.required = True
+
+[template]
+pre_render = bobtemplates.itpp.hooks:pre_render_copyright
```
Comment (KolushovAlexandr) [inline bobtemplates/itpp/copyright/.mrbob.ini]:
```suggestion
pre_render = bobtemplates.itpp.hooks:pre_render_copyright

```

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]
* Properly update readme file. 
* When it asks for category it provides 'pos' as an example and then types it into the manifest as 'pos' not as "Point of Sale"
* smhw got this "https://apps.odoo.com/apps/modules/{VERSION}/bob_technical_name/"
* ```__pycache__``` dir appeared in modules repository
* no empty line at the end of changelog and index.rst files
* there is no a closing bracket at the last line of index.rst file
* icon file is broken
* if I typed version as '12' it pastes 12 everywhere, maybe make handler for those cases?
* 
```
mrbob bobtemplates.itpp.odoo:model
Traceback (most recent call last):
  File "/usr/local/bin/mrbob", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/mrbob/cli.py", line 155, in main
    defaults=defaults)
  File "/usr/local/lib/python3.5/dist-packages/mrbob/configurator.py", line 172, in __init__
    self.post_render = [resolve_dotted_func(f) for f in self.templateconfig.get('post_render', '').split()]
  File "/usr/local/lib/python3.5/dist-packages/mrbob/configurator.py", line 172, in <listcomp>
    self.post_render = [resolve_dotted_func(f) for f in self.templateconfig.get('post_render', '').split()]
  File "/usr/local/lib/python3.5/dist-packages/mrbob/configurator.py", line 45, in resolve_dotted_func
    module = import_module(module_name)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'bobtemplates.odoo'
```
Instructions unclear got a duck stuck in a toaster

**Pull request merged**